tests/test_flows/test_flow.py

```diff
@@ -153,6 +153,23 @@ def test_publish_flow(self):
         flow.publish()
         self.assertIsInstance(flow.flow_id, int)
 
+    def test_semi_legal_flow(self):
```

Comment:
While that might be the case, it contains two distinguishable forms of bagging. 
- Bagging(Bagging(J48))
- Bagging(J48)

Therefore, OpenML will be able to set the parameters of the individual components correct at any run, and there is no problem